templates/patterns/tag/tag.ui_patterns.yml

```diff
@@ -0,0 +1,19 @@
+tag:
+  label: "Tag"
+  description: "The tag component is used to display structured information."
+  fields:
+    tag_type:
+      type: "text"
+      label: "Type"
+      description: "Type of tag (link, button, removable)"
+      preview: "link"
+    path:
+      type: "text"
+      label: "Path"
+      description: "Tag path"
+      preview: "#"
```

Comment:
If this is the tag link URL I'd call it `url` instead of `path` as it might contain a full URL, other than a relative path like `/something`.